test/manual/analytics-batching.amp.html

```diff
@@ -41,7 +41,7 @@
 <p>
 Integer in felis at lacus mattis facilisis. Curabitur tincidunt, felis porttitor mollis finibus, tortor elit elementum dolor, vel vulputate lorem dui id ante. Vivamus in velit at lectus blandit gravida vitae quis arcu. Nam et magna magna. Fusce condimentum diam lacus, ac ullamcorper purus malesuada eu. Mauris ullamcorper elit et venenatis faucibus. Nullam lobortis molestie purus quis pellentesque. Sed at libero id nisi rhoncus tincidunt. Praesent vestibulum vehicula tristique. Etiam rutrum, nunc id porta interdum, nulla nisi molestie leo, at fermentum justo dolor at lorem. Duis in egestas sapien.
 </p>
-<amp-analytics type="_ping_">
```

Comment:
Why this change?

Reply:
we don't have type `_ping_` : ) Remove it to fix the test page.

Reply:
it would be clearer if you called it something like `nonexistent-vendor`, so readers could tell it wasn't supposed to work

Reply:
This is my go-to manual test file : ) Didn't meant to test nonexistent vendor. But it's a good point that we can introduce a test to cover invalid vendors.